### ISSUE TITLE
Erro com o loader.js

### ISSUE THREAD

**Issue by hulaneoTi** [IncursioHack/ESP-Game-Server#1]
Após escolher o jogo a página para de carregar e tenho a seguinte mensagem no console:

"Uncaught SyntaxError: expected expression, got '<'
loader.js:2:1"

**Comment by nimx86** [comment]
experiencing the same issue when loading a .gba file, didn't test for the other ones

**Comment by tengotengo** [comment]
yeah. /data/loader.js returns 302 and redirects to / which is an HTML page, so we're trying to parse HTML as JS.

**Comment by tengotengo** [comment]
OK so the instructions were unclead. You have to copy everything from https://github.com/IncursioHack/ESP-Game-Server/tree/main/ESPGameServer/data  to your SD card, in ESPGameServer directory. 
You can skip the cores you don't need, they're too big.
